docs/ref/migration-operations.txt

```diff
@@ -232,6 +232,25 @@ manually creating a column, you should pass in a list containing an ``AddField``
 operation here so that the autodetector still has an up-to-date state of the
 model (otherwise, when you next run ``makemigrations``, it won't see any
 operation that adds that field and so will try to run it again).
+For example::
+
+    migrations.RunSQL(
+        "ALTER TABLE musician "
```

Comment:
could add an `app_label` to the table name.
could we simplify the example by adding a single field?